rpm/gratia-probe.spec

```
@@ -14,6 +14,8 @@ BuildRequires:      gcc-c++
 BuildRequires:      python3
 %endif
 
+BuildRequires:      git
```

```suggestion
%if 0%{?rhel} < 8
BuildRequires:      git
%else
BuildRequires:      git-core
%endif
```
Gives you a somewhat smaller buildroot (which means a faster build).